packages/hooks/package.json

```diff
@@ -27,12 +27,12 @@
     "kaioken": "^0.31.0"
```

Comment:
can you also update the peerDeeps version number for kaioken aswell, assuming this is necessary

Reply:
Done 👌